src/app/test-record/defects/defects.component.spec.ts

```diff
@@ -36,4 +39,14 @@ describe('DefectsComponent', () => {
 
     expect(fixture).toMatchSnapshot();
   });
+
+  it('should match snapshot if test type is among specialist tests', () => {
```

Comment:
`should generate test type for specialist tests`. the snapshot file is your outcome